zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -787,7 +787,7 @@ private static void sendToKafka(Collection collection) throws IOException {
                 .data("Reviewed-uris", reviewedUris)
                 .data("publishing", true)
                 .log("converted reviewed URIs for kafka event");
-        sendMessage (collection, reviewedUris, "Reviewed-uris");
+        sendMessage (collection, reviewedUris, "legacy");
```

Comment:
Same here, `public static final String LEGACYCONTENTFLAG = "legacy";`

```suggestion
        sendMessage (collection, reviewedUris, LEGACYCONTENTFLAG);
```

Maybe there is a better variable name then `LEGACYCONTENTFLAG`